### ISSUE TITLE
Unexpected additional windows with domain auto-popup

### ISSUE THREAD

**Issue by grahamperrin** [aqt/webext-popup#30]
Spun off from https://github.com/aqt/webext-popup/issues/12#issuecomment-423824238 for PopUp 1.3.7 with a heavily-extended Waterfox profile. 

Re: https://redd.it/93x02v Extension Conflict Troubleshooter seemed to positively identify a conflict with legacy experimental Load Tabs Progressively (<http://web.archive.org/web/20171030192628/https://addons.mozilla.org/en-US/firefox/addon/load-tabs-progressively-fixed/> in the Wayback Machine). 

However after re-enabling my usual set of extensions _except_ Load Tabs Progressively, the issue persisted. 

Next disabled: [Load Background Tabs Lazily](https://addons.mozilla.org/addon/load-background-tabs-lazily/), the issue persisted. 

----

Rewind a little. 

From <https://github.com/aqt/webext-popup/issues/5#issuecomment-423772534>: 

> …  I don't really want to add a fix to do this automatically though since it would look sketchy with a window suddenly opening and closing.

For what it's worth, that (visible automated opening and automated closing) **is** what I saw at least twice during the Extension Conflict Troubleshooter routine. 

At another point in the routine I found _multiple_ additional windows, **not** automatically closed. Here, three more than expected:  

![2018-09-23 16 41 07 waterfox](https://user-images.githubusercontent.com/192271/45930261-e16de080-bf55-11e8-9b75-ac109b888763.png)

Here, one more than expected (to the left) and its title relates to the window to its right: 

![2018-09-23 16 53 07 waterfox](https://user-images.githubusercontent.com/192271/45930295-2c87f380-bf56-11e8-9b91-499665854bec.png)

----

I have a shed load of extensions, I guess that this issue might be difficult to make _consistently_ reproducible, so feel free to simply close it issue as non-fixable. 

Thanks

**Comment by aqt** [comment]
If you can find out which combination of extensions can reliably trigger the issue I can troubleshoot further. If I can't reproduce it it will be too difficult to solve and I'll have to close this issue as wontfix.

The only case I know of that would cause these blank windows is when restoring a popup to a tab, which I re-added a fix for as you saw (new fix is a bit cleaner compared to the old one but the logic *is* changed, which could be a possible suspect if it happens in this case).

**Comment by aqt** [comment]
I have been experiencing this as well recently, however it's inconsistent and only happens on my work laptop. I haven't seen it at home with regular use - haven't yet tried to forcefully reproduce it.

I will prioritize this issue after #23

**Comment by grahamperrin** [comment]
I had completely forgotten this issue, which probably means that months have passed since it was last seen. 

Let's close, it can be reopened if necessary.
